ass2-task/sass/pages/_home.scss

```diff
@@ -0,0 +1,786 @@
+      padding: 0 10px;
+      text-align: center;
+      width: 250px;
+
+      img {
+        margin-top: 12px;
+        width: 180px;
+      }
+    }
+
+    .menu {
+      margin: 20px 10px;
+
+      i {
+        font-size: 25px;
+      }
+    }
+
+    .navbar-options {
+      float: right;
+      line-height: 65px;
+
+      li {
+        display: inline-block;
+        padding: 0 15px;
+
+        a {
+          color: $white;
+          display: block;
+
+          i {
+            font-size: 25px;
+            position: relative;
+            text-align: center;
+            vertical-align: middle;
+          }
+
+          span {
+            background-image: url('../img/man.jpg');
+            background-position: center;
+            background-size: cover;
+            border-radius: 50%;
+            display: inline-block;
+            height: 30px;
+            vertical-align: middle;
+            width: 30px;
+          }
+
+          p {
+            display: inline-block;
+            margin-left: 10px;
+          }
+        }
+      }
+
+      .badge-envelope::after {
+        background-color: $green;
+        border: 1px solid $green;
+        content: '4';
+      }
+
+      .badge-bell::after {
+        background-color: $orange;
+        border: 1px solid $orange;
+        content: '19';
+      }
+
+      .badge-flag::after {
+        background-color: $red;
+        border: 1px solid $red;
+        content: '10';
+      }
+    }
+  }
+}
+
+//===============Navbar Ends==========//
+
+//==============Sidebar==============//
+
+.sidebar {
+  background-color: $cinder;
+  color: $light-grey;
+  height: 100%;
+  overflow-y: auto;
+  padding: 20px 0;
+  position: fixed;
+  width: 250px;
+
+  .user-info {
+    padding: 0 10px;
+
+    .user-img {
+      background-image: url('../img/man.jpg');
+      background-position: center;
+      background-size: cover;
+      border-radius: 50%;
+      display: inline-block;
+      height: 50px;
+      width: 50px;
+    }
+
+    .user-name {
+      display: inline-block;
+      margin-left: 20px;
+      vertical-align: top;
+
+      h2 {
+        margin-bottom: 7px;
+      }
+
+      i {
+        color: $green;
+        margin-right: 10px;
+      }
+    }
+
+    .search-container {
+      background-color: $input-black;
+      border-radius: 4px;
+      margin: 20px 0 10px;
+      padding: 6px 10px;
+
+      input {
+        background-color: $input-black;
+        width: 90%;
+      }
+    }
+  }
+
+  .sidebar-header {
+    input[type='search'] {
+      background-color: $dark-black;
+      color: $white;
+      font-size: 14px;
+      line-height: 30px;
+      padding: 10px;
+      text-transform: uppercase;
+      width: 100%;
+    }
+  }
+  
+  .label-menu {
+    margin-bottom: 50px;
+  }
+
+  .navigation-menu,
+  .label-menu {
+    a {
+      color: $font-black;
+      display: block;
+      line-height: 36px;
+      margin: 5px 0;
+      padding: 5px 20px;
+
+      i:nth-child(1) {
```

Comment:
@priyanka0901 You should have added two class for the styling `i` tag left and `i` tag right instead of nth-child. nth-child can break anytime if I have to add the icon which is not in left or right.